src/composables/useNetwork.ts

```diff
@@ -0,0 +1,31 @@
+import { ref } from 'vue';
+
+export type NetworkId = 1 | 3 | 4 | 5 | 42 | 137 | 42161;
```

Comment:
For now I'm keeping this `NetworkId`, but I believe we can end up just using the enum.

Reply:
It is my understanding that Balancer is deployed on all TestNets, and that we just need to support it on the frontend.
The Rinkeby/Ropsten networks are actually used in the code for Matcha service / Gnosis endpoint.

I guess its possible to remove it until we support it, but also feels like there is no harm for now?

Reply:
Yeah fine to leave it, just checking.